pkg/tagger/collectors/kubernetes_metadata_mapper.go

```diff
@@ -18,10 +19,24 @@ import (
 )
 
 func (c *KubeMetadataCollector) getTagInfos(pods []*kubelet.Pod) []*TagInfo {
+	var err error
+	metadataByNsPods := apiv1.NewNamespacesPodsStringsSet()
+	if config.Datadog.GetBool("cluster_agent.enabled") {
```

Comment:
getTagInfos is getting called fairly often, what do you think about moving this query to Viper in the  `KubeMetadataCollector` ? When looking at pprofs I often see a lot of calls to Viper.

Reply:
I also removed the `config.Datadog.GetBool()` from other place in `KubeMetadataCollector`